packages/sdk-components-react/src/code-text.tsx

```diff
@@ -1,14 +1,35 @@
-import { type ElementRef, type ComponentProps, forwardRef } from "react";
+import {
+  type ElementRef,
+  type ComponentProps,
+  forwardRef,
```

Comment:
😠

Reply:
what about it?

Reply:
Wanna get rid of forwardRef already 😄

Reply:
I totally didn't realize we already are on the version that doesn't require it lol